packages/shared/src/components/buttons/QuaternaryButton.tsx

```diff
@@ -24,6 +25,7 @@ function QuaternaryButtonComponent<TagName extends AllowedTags>(
     reverse,
     responsiveLabelClass,
     tag = 'button',
+    labelClassName,
```

Comment:
Why not opt in for classname as an object? Or too many changes that way?

Reply:
We should really get rid of this button comp anyway 🥲

Reply:
agree

Reply:
because I did not want to refactor all other usages

Reply:
@nensidosari as to downvote/block actions I think we should fix it separate since this only adds the button and reuses logic already used for lists from before?

As to height issue, I will fix it, good catch 👌

Reply:
@capJavert Makes sense 🙌🏻 I'll create a ticket for what I found that are not relevant here

Reply:
Fixed height in a9869501031b98c4099c343c5ce59b5e199756eb, I am not sure why `max-h` was used but from testing looks good without it and tags panel just expands into card container. 
<img width="714" alt="image" src="https://github.com/user-attachments/assets/d4ca2e7e-2580-4c87-b100-0e76d37316a3">